Update kafka producer example (#1298)
wso2/docs-ei: en/micro-integrator/docs/references/connectors/kafka-connector/kafka-connector-producer-example.md

Given below is a sample scenario that demonstrates how to send messages to a Kafka broker via Kafka topics.The publishMessages operation allows you to publish messages to the Kafka brokers via Kafka topics.

## What you'll build

> Given below is a sample API that illustrates how you can connect to a Kakfa broker with the `init` operation and then use the `publishMessages` operation to publish messages via the topic.It exposes Kakfa functionalities as a restful service. Users can invoke the API with HTTP/HTTPs with required information.

Reviewer: ```suggestion
Given below is a sample API that illustrates how you can connect to a Kakfa broker with the `init` operation and then use the `publishMessages` operation to publish messages via the topic. It exposes Kakfa functionalities as a RESTful service. Users can invoke the API using HTTP/HTTPs with the required information.
```

Author: Updated.

> API has the context `/publishMessages`.It will publish messages via the topic to the kafka server.

Reviewer: ```suggestion
API has the context `/publishMessages`. It will publish messages via the topic to the Kafka server.
```

Author: Updated.

The following diagram illustrates all the required functionality of the Kafka service that you are going to build.

<img src="/assets/img/connectors/KafkaConnectorPublishMessage.png" title="KafkaConnector" width="800" alt="KafkaConnector"/>

## Configure the connector in WSO2 Integration Studio

Follow these steps to set up the ESB Solution Project and the Connector Exporter Project.

{!references/connectors/importing-connector-to-integration-studio.md!}

> 1. Right click on the created ESB Solution Project and select, -> **New** -> **Rest API** to create the REST API.

Reviewer: ```suggestion
1. Right click on the created Integration Project and select, -> **New** -> **Rest API** to create the REST API.
```

Author: Updated.

            </inSequence>
            <outSequence/>
            <faultSequence/>
           </resource>
    </api>
    ```

> Now we can export the imported connector and the API into a single CAR application. CAR application is the one we are going to deploy to server runtime. 

Reviewer: ```suggestion
Now we can export the imported connector and the API into a single CAR application. The CAR application will be deployed during server runtime. 
```

Author: Updated.

{!/references/connectors/exporting-artifacts.md!}

## Deployment

Follow these steps to deploy the exported CApp in the Enterprise Integrator Runtime. 

{!references/connectors/deploy-capp.md!}
    
## Testing

>    **Create a topic**:

Reviewer: Why have we tabbed this? Tab can be removed for this section.

Author: Tabs removed.
